app/assets/stylesheets/components/_modal.scss

```diff
@@ -35,14 +35,14 @@
     position: relative;
 
     &.key-badge::before {
-      background-image: url('p-key.svg');
+      background-image: url('personal-key/personal-key.svg');
```

Comment:
Do we need to set `background-size` here, or resize the SVG? Currently it looks like it is being clipped because the size of the image is larger than 88px:

![image](https://user-images.githubusercontent.com/1779930/191511619-ef4d4a4b-6c02-4234-889c-cc1b12880c1b.png)

Reply:
Something must have happened when I re-exported from Figma. I'll take another pass at all of them and see what's up